docs/xamarin-forms/user-interface/images.md

```diff
@@ -101,6 +101,8 @@ Android alternate resolution images should be placed in [specially-named directo
 
 [![Android Multiple-Resolution Image Location](images-images/xs-highdpisolution-sml.png "Android Multiple-Resolution Image Location")](images-images/xs-highdpisolution.png#lightbox "Android Multiple-Resolution Image Location")
 
+UWP and Windows Phone image file names [can be suffixed with `.scale-xxx` before the file extension](https://docs.microsoft.com/en-us/windows/uwp/app-resources/images-tailored-for-scale-theme-contrast), where `xxx` is the percentage of scaling applied to the asset, e.g. **myimage.scale-200.png**. Images can then be referred to in code or XAML without the scale modifier, e.g. just **myimage.png**. The platform will select the nearest appropriate asset scale based on the display's current DPI.
```

Comment:
remove the locale from the URL, so translations are redirected to the correct language, ie `https://docs.microsoft.com/windows/uwp/app-resources/images-tailored-for-scale-theme-contrast/`